weblab/static/gulpfile.js

```diff
@@ -4,13 +4,14 @@ var watchify = require('watchify');
 var browserify = require('browserify');
 var source = require('vinyl-source-stream');
 var buffer = require('vinyl-buffer');
-var log = require('fancy-log');
 var sourcemaps = require('gulp-sourcemaps');
-var assign = require('lodash.assign');
+var es = require('event-stream');
 
 var config = {
   js: {
-    src: './js/main.js',
+    src: [
+      './js/main.js',
```

Comment:
The name of the original file should map through to the name of the bundled minified js. To use it, yes that would be correct. I think the way the templates do that might need to change a little (e.g. take that line out of the standard head.html and bring it into base.html with a block around it so it can be overridden

something like this:

```
{% include "includes/head.html" %}
{% block page_js %}
  <script src="{% static 'build/js/main.js' %}"></script>
{% endblock page_js %}
```

and then the `page_js` block could be overridden in templates